docs/Installing.asciidoc

```diff
@@ -268,14 +268,15 @@ according to
 [[authentication]]
 === User authentication
 
-OpenQA supports two different authentication methods - OpenID (default) and
-Fake. See `auth` section in `/etc/openqa/openqa.ini`.
+OpenQA supports three different authentication methods - OpenID (default),
+Fake (for development) and OAuth 2.0 (currently limited to GitHub).
+See `auth` section in `/etc/openqa/openqa.ini`.
 
 [source,ini]
 --------------------------------------------------------------------------------
 [auth]
 # method name is case sensitive!
-method = OpenID|Fake
+method = OpenID
```

Comment:
I prefer to keep all options listed here

Reply:
This is an example config, though? Lists separated by pipes are not supported... :confused:

Reply:
Well, that's the problem with "example config". To me it was pretty clear that it says to write `method = OpenID` or `method = Fake` or `method = OAuth2` but now I see it might not be that obvious :)

Reply:
I added `[auth]` to the config examples in the following sections now, to make it even clearer what the respective values are.